### DISCUSSION TITLE
喜欢游戏里的那种温馨和惬意

### DISCUSSION THREAD

**Discussion by Micawber-jock** [share121/inter-knot#1341] [其他]
![20240721133238](https://github.com/user-attachments/assets/c04680e1-8462-4fea-a4dc-537bc79adf1b)
追求等级，速通和过任务的途中不妨看看身边的景色，比如吃碗面，拍拍照，和npc聊一聊，约代理人出来玩，代入感强了这就会像是虚拟的家，前两年玩原神的时候就会仿佛置身提瓦特了解里面的人文地理，但后来就没有那份热爱

**Comment by gaojiaobei** [comment]
‘游戏里的一花一草都是游戏啊

**Comment by ZhuYuan-148** [comment]
治安局发誓会守护好这份和谐，保障各位市民的安全！
